typescript/sdk/src/providers/transactions/transformer/TxTransformerTypes.ts

```diff
@@ -1,3 +1,3 @@
 export enum TxTransformerType {
-  ICA = 'Interchain Account',
+  INTERCHAIN_ACCOUNT = 'interchainAccount',
```

Comment:
is this exported outside of the SDK? would this be a breaking change?

Reply:
Yes, and yes in isolation it is breaking. It's currently only consumed by my submit changes on CLI-side, which is not exposed via a command yet. This PR makes that change CLI-side [here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/27be8100d7db710277cfb166f3e1219f146907e0/typescript/cli/src/submit/submit.ts#L78)